### PR TITLE
Add Input ~ input example

### PR DESCRIPTION
**Pull request by nnupoor**
Issue #817 example added.

### CODE REVIEW

File: live-examples/html-examples/input/css/input.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+@font-face {
+  font-family: 'Fira Sans';
+  src: local('FiraSans-Regular'),
+  url('/media/fonts/FiraSans-Regular.woff2') format('woff2');
```
Comment (schalkneethling) [inline live-examples/html-examples/input/css/input.css]: Nit: Code Style: indent with 4 spaces

File: live-examples/html-examples/input/css/input.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+legend {
+  background-color: #000;
+  color: #ffffff;
+  padding: 3px 6px;
```
Comment (schalkneethling) [inline live-examples/html-examples/input/css/input.css]: Nit: Code Style: indent with 4 spaces

File: live-examples/html-examples/input/css/input.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
+@font-face {
+  font-family: 'Fira Sans';
+  src: local('FiraSans-Regular'),
+  url('/media/fonts/FiraSans-Regular.woff2') format('woff2');
+}
+
+legend {
+  background-color: #000;
+  color: #ffffff;
```
Comment (schalkneethling) [inline live-examples/html-examples/input/css/input.css]: Nit: Code Style: Prefer short form when applicable `#ffffff` => `#fff`

File: live-examples/html-examples/input/css/input.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+label {
+    width: 43%;
+}
+
+input {
+  margin: 0.5rem 0;
+  padding: 0.5rem;
+  border-radius: 4px;
+  border: 1px solid #ddd;
+  outline: none;
```
Comment (schalkneethling) [inline live-examples/html-examples/input/css/input.css]: Nit: Code Style: indent with 4 spaces. Do not remove outline as it negatively impacts accessibility, unless there is alternate styling provider for the `active` and `focus` states.

File: live-examples/html-examples/input/css/input.css [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+label {
+    display: inline-block;
+    font-size: .8rem;
+}
+
+input:invalid + span:after {
+  content: '✖';
+  color: red;
+  padding-left: 5px;
```
Comment (schalkneethling) [inline live-examples/html-examples/input/css/input.css]: Nit: Code Style: indent with 4 spaces

File: live-examples/html-examples/input/css/input.css [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+input:valid + span:after {
+  content: '✓';
+  color: green;
+  padding-left: 5px;
```
Comment (schalkneethling) [inline live-examples/html-examples/input/css/input.css]: Nit: Code Style: indent with 4 spaces

File: live-examples/html-examples/input/css/input.css [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.warning {
+  font-size: 0.65rem;
+  color: #e67d09;
```
Comment (schalkneethling) [inline live-examples/html-examples/input/css/input.css]: Nit: Code Style: indent with 4 spaces

File: live-examples/html-examples/input/css/input.css [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.submit {
+  width: 92%;
+  margin: 0 auto;
```
Comment (schalkneethling) [inline live-examples/html-examples/input/css/input.css]: Nit: Code Style: indent with 4 spaces

File: live-examples/html-examples/input/input.html [hunk 9]
```diff
@@ -0,0 +1,24 @@
+<fieldset>
+    <legend>Input</legend>
+    <form>
+        <label for="display-name"> Name:
+            <span class="warning">*(Allows only alphabets.)</span> 
+        </label>
+        <input type="text" name="display-name" 
+               pattern="[A-Za-z\s]+"
+               maxlength="5" minlength="2" value="Aa" required>
+        <span class="validity"></span>
```
Comment (schalkneethling) [inline live-examples/html-examples/input/input.html]: Is this element being used?

File: live-examples/html-examples/input/input.html [hunk 10]
```diff
@@ -0,0 +1,24 @@
+<fieldset>
+    <legend>Input</legend>
+    <form>
+        <label for="display-name"> Name:
+            <span class="warning">*(Allows only alphabets.)</span> 
+        </label>
+        <input type="text" name="display-name"
```
Comment (schalkneethling) [inline live-examples/html-examples/input/input.html]: This element is missing an `id` attribute with a value that matches the `for` attribute of the associated `label` element.

File: live-examples/html-examples/input/input.html [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+        <label for="display-name"> Name:
+            <span class="warning">*(Allows only alphabets.)</span> 
+        </label>
+        <input type="text" name="display-name" 
+               pattern="[A-Za-z\s]+"
+               maxlength="5" minlength="2" value="Aa" required>
+        <span class="validity"></span>
+
+        <label for="readonly-ip">ReadOnly:</label>
+        <input type="text" name="readonly-ip"
```
Comment (schalkneethling) [inline live-examples/html-examples/input/input.html]: This element is missing an `id` attribute with a value that matches the `for` attribute of the associated `label` element.

File: live-examples/html-examples/input/input.html [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+               placeholder="I'm read only." readonly />
+
+        <label for="disabled-ip">Disabled:</label>
+        <input type="text"
```
Comment (schalkneethling) [inline live-examples/html-examples/input/input.html]: This element is missing an `id` attribute with a value that matches the `for` attribute of the associated `label` element.

File: live-examples/html-examples/input/input.html [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+               name="disabled-ip"
+               value="I am disabled"
+               disabled />
+        <br/>
```
Comment (schalkneethling) [inline live-examples/html-examples/input/input.html]: Code style: Do not break between form element with `br`, instead wrap each `label/input` group in a `div`

File: live-examples/html-examples/input/input.html [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+        <input type="submit" class="submit" value="Submit"/>
```
Comment (schalkneethling) [inline live-examples/html-examples/input/input.html]: Nit: Code style: Space before closing `/>`

File: live-examples/html-examples/input/css/input.css [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+    content: '✓';
+    color: green;
+    padding-left: 5px;
+}
+
+.warning {
+    font-size: .65rem;
+    color: #e67d09;
+}
+
```
Comment (nnupoor) [inline live-examples/html-examples/input/css/input.css]: @schalkneethling Removed outline: none, 0.5rem => .5rem, spacing fixes, #ffffff => #fff in CSS

File: live-examples/html-examples/input/input.html [hunk 16]
```diff
@@ -0,0 +1,30 @@
...
+    <legend>Input</legend>
+    <form>
+        <div>
+            <label for="display-name"> Name:
+                <span class="warning">*(Allows only alphabets.)</span> 
+            </label>
+            <input type="text" id="display-name" name="ip-display"
+                   pattern="[A-Za-z\s]+"
+                   maxlength="5" minlength="2" value="Aa" required />
+            <span></span>
```
Comment (nnupoor) [inline live-examples/html-examples/input/input.html]: Closed tags, added id for inputs for accessibility. Removed <br> tags.

File: live-examples/html-examples/input/input.html [hunk 17]
```diff
@@ -0,0 +1,30 @@
+<fieldset>
+    <legend>Input</legend>
+    <form>
+        <div>
+            <label for="display-name"> Name:
+                <span class="warning">*(Allows only alphabets.)</span>
```
Comment (schalkneethling) [inline live-examples/html-examples/input/input.html]: Nit: Please change this to read "*(Allows only letters.)"

**Comment by nnupoor** [comment]
@schalkneethling Updated alphabets => letters

**Pull request merged**